lib/toFixed.js

```diff
@@ -1,5 +1,16 @@
 'use strict';
 
+/**
+ * Strip decimal points from a number.
+ *
+ * @since v0.0.1
+ * @param {Number} num
+ * @return {Number}
+ * @example
+ *
+ *   {{toFixed 19.99}} //=> 19
+ */
+
```

Comment:
I think this is backwards, actually. It should _add_ the decimals. So `1` passed in would yield `"1.00"`.